nemo/collections/nlp/models/intent_slot_classification_model.py

```diff
@@ -224,7 +224,6 @@ def _setup_dataloader_from_config(self, cfg: DictConfig):
                  of Intents and Slots files.'
             )
 
-        # TODO: check that file exists
         dataset = IntentSlotClassificationDataset(
```

Comment:
Do you check it dataset file exists?

Reply:
Yes, I do in rows before that. I just removed old TODO comment here.